lib/email_alert_api/config.rb

```diff
@@ -23,6 +23,10 @@ def notify
       @notify ||= notify_environment_config.symbolize_keys.freeze
     end
 
+    def email_service_provider
+      @email_service_provider ||= email_service_provider_config.symbolize_keys.fetch(:provider).freeze
```

Comment:
Is it worth turning this into a symbol?

Reply:
maybe not - was originally exposing the whole yaml file here as with the rest of the class, but decided it wasn't worth it yet.

Reply:
Hmm, yeah, that makes sense if we may expose more in the future.

Reply:
I've changed it anyway - no big deal to change it back if we need to.